File: codeforces/3.team/team.cpp
#include <iostream>

/* Author: José Rodolfo (jric2002) */

using namespace std;

int main() {
  unsigned short int n;
  cin >> n;
  if (n >= 1 && n <= 1000) {
    unsigned short int probability;
    unsigned short int solutions = 0;
    bool petya, vasya, tonya;
    for (unsigned short int r = 0; r < n; r++) {
      cin >> petya >> vasya >> tonya;
      probability = petya + vasya + tonya;
      if (probability >= 2) {
        solutions++;
      }
    }
    cout << solutions << endl;
  }
  return 0;
}

File: codeforces/3.team/team.py
# Author: José Rodolfo (jric2002)
n = int(input())
if (n >= 1 and n <= 1000):
  petya = vasya = tonya = 0
  probability = 0
  solutions = 0
  for r in range(0, n):
    input_data = input().split(" ")
    input_data = list(map(int, input_data))
    petya = input_data[0]
    vasya = input_data[1]
    tonya = input_data[2]
    probability = petya + vasya + tonya
    if (probability >= 2):
      solutions += 1
  print(solutions)